docker/gaffer-kerberos/gaffer-integration-tests-krb/accumulo-int-entrypoint.sh

```diff
@@ -14,8 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Wait a minute for Accumulo to be started and working
-sleep 60
+# Wait for Accumulo to be started and working (uses same approach as compose healthcheck)
+until cat /proc/net/tcp | grep 270F; do
```

Comment:
Maybe could add a max counter like 20 attempts before it fails?

Reply:
The docker compose check does this and if that times out the container will be stopped. That will stop this script so not really a need.